packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -724,6 +724,43 @@ function updateTsLintConfig(): Rule {
   };
 }
 
+function updateRootTsConfig(): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    const tsConfigPath = '/tsconfig.json';
+    const buffer = host.read(tsConfigPath);
+    if (!buffer) {
+      return;
+    }
+
+    const tsCfgAst = parseJsonAst(buffer.toString(), JsonParseMode.Loose);
+    if (tsCfgAst.kind != 'object') {
+      throw new SchematicsException('Invalid tsconfig. Was expecting an object');
+    }
+
+    const compilerOptionsAstNode = findPropertyInAstObject(tsCfgAst, 'compilerOptions');
+    if (!compilerOptionsAstNode || compilerOptionsAstNode.kind != 'object') {
+      throw new SchematicsException(
+        'Invalid tsconfig "compilerOptions" property; expected an object.',
+      );
+    }
+
+    if (
+      findPropertyInAstObject(compilerOptionsAstNode, 'baseUrl') &&
+      findPropertyInAstObject(compilerOptionsAstNode, 'module')
+    ) {
+      return host;
+    }
+
+    const compilerOptions = compilerOptionsAstNode.value;
+    compilerOptions.module = compilerOptions.module || 'es2015';
+    compilerOptions.baseUrl = compilerOptions.baseUrl || './';
```

Comment:
If this is already set and not a variation of the current directory (e.g., "", ".", or "./") then a warning should probably be shown indicating that generated library path mapping will not work.